src/util/Constants.js

```diff
@@ -49,6 +49,10 @@ exports.Endpoints = {
         if (dynamic && hash.startsWith('a_')) format = 'gif';
         return makeImageUrl(`${root}/avatars/${userId}/${hash}`, { format, size });
       },
+      GuildMemberAvatar: (guildId, memberId, hash, format = 'webp', size, dynamic = false) => {
+        if (dynamic) format = hash.startsWith('a_') ? 'gif' : format;
```

Comment:
```suggestion
        if (dynamic && hash.startsWith('a_')) format = 'gif';
```
So it matches the others.

Reply:
Right, forgot I changed that